app/views/new_filters/_area_provider_filter.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="app-filter__group" data-qa="filters__area_and_provider">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+    <h3 class="govuk-fieldset__heading">
+      <% if request.parameters['query'] %>
+        Provider
+      <% else %>
+        Area
```

Comment:
Until we’ve moved to the new location model, let’s stick to saying ‘location’ for now.

```suggestion
        Location
```

Reply:
👍